Spigot-Server-Patches/0696-Can-Fly-Fall-Damage.patch

```diff
@@ -0,0 +1,62 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TreyRuffy <TreyRuffy@users.noreply.github.com>
+Date: Sat, 13 Mar 2021 12:59:14 -0700
+Subject: [PATCH] Can Fly Fall Damage
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/player/EntityHuman.java b/src/main/java/net/minecraft/world/entity/player/EntityHuman.java
+index ec0956a98c133bcd3d4f92f696c667eab6ff98f1..fdc469d9d9f708e67d21757ac2227da261a7b532 100644
+--- a/src/main/java/net/minecraft/world/entity/player/EntityHuman.java
++++ b/src/main/java/net/minecraft/world/entity/player/EntityHuman.java
+@@ -169,6 +169,7 @@ public abstract class EntityHuman extends EntityLiving {
+     public EntityFishingHook hookedFish;
+     // Paper start
+     public boolean affectsSpawning = true;
++    public boolean enableFallDamageWhileAllowFlight = false;
```

Comment:
This won't be persisted across server restarts, players leaving and joining since its not saved anywhere. Intended?

Reply:
I made it that way but I am happy to save it if you believe it would be beneficial.

Reply:
I have added a configuration option for this!